src/main/java/frc/robot/Controls.java

```diff
@@ -44,10 +44,12 @@ public class Controls {
   private BranchHeight branchHeight = null;
 
   // Swerve stuff
-  private double MaxSpeed =
+  public static double MaxSpeed =
       RobotType.getCurrent() == RobotType.BONK
```

Comment:
This doesn't look like it needs to be `public` as I don't find anything outside this class that uses it. Also, it should be `final` since it never changes after initialization.